src/style.css

```diff
@@ -0,0 +1,74 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-weight: bold;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.search {
+  margin-top: 20px;
+  position: relative;
+}
+
+.search__bar {
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  color: #3d4e61;
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.search::after {
+  position: absolute;
+  content: "";
+  display: block;
+  width: 19px;
+  height: 19px;
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  left: 26px;
+  top: 25px;
+}
+
+.search--small.search::after {
+  width: 11px;
+  height: 11px;
+  left: 13px;
+  top: 15px;
+}
+
+.search__bar::placeholder {
+  color: inherit;
+}
+
+.search__bar:focus {
+  outline: none;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  font-weight: bold;
+}
+
+.search__bar:hover {
+  box-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.search--small .search__bar {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+}
```

Comment:
please use selectors by class. and why did you giving height 42px here when you have [this](https://github.com/mate-academy/layout_search-bar-airbnb/pull/3738/files#diff-1104bdfdc4d3b01a85d7af489598121125db6bc016db2a48a083ef654c97b0f5R24)?